app/routes/schemas/debt-type.js

```diff
@@ -0,0 +1,24 @@
+const Joi = require('joi')
+
+const typeOfDebts = ['irr', 'admin']
```

Comment:
These are what are saved down as debtType is the database, are these correct? I've seen `Irregular` and `Administrative` used elsewhere